.travis.yml

```diff
@@ -17,9 +17,7 @@ install:
 - travis_retry npm install
 script:
 - grunt
-- grunt intern:node --combined
-- grunt intern:saucelabs --combined
-- grunt remapIstanbul:ci
+- grunt intern:browserstack --test-reporter
```

Comment:
We're switching from saucelabs to browserstack here. Is that intentional?

Reply:
Yes